{"uuid": "13994e24-4f47-433b-9a33-249c70f6b30e", "children": ["de551e57-e209-4a75-a8b8-450ccf788dee"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "Exception: 无该校验方式：no_check\n", "trace": "  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/nali/anaconda3/lib/python3.7/site-packages/_pytest/fixtures.py\", line 792, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/nali/AutoApiTest/conftest.py\", line 28, in login\n    api_send_check(request.param, case_dict, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/api_send_check.py\", line 33, in api_send_check\n    check_result.check_result(test_name, check_data, code, data, case_path, relevance)\n  File \"/Users/nali/AutoApiTest/common/unit/check_result.py\", line 128, in check_result\n    raise Exception(\"无该校验方式：{}\".format(case[\"check_type\"]))\n"}, "steps": [{"name": "POST请求接口", "status": "passed", "attachments": [{"name": "请求接口：", "source": "12114033-f06e-4925-b9e0-d05d92954e74-attachment.attach"}, {"name": "用例描述：", "source": "cfcede4e-3e05-4a69-95c4-934923b61d4c-attachment.attach"}, {"name": "请求地址", "source": "98fd6e0a-4cb6-417a-aee4-688f78a15ff2-attachment.attach"}, {"name": "请求头", "source": "24d58b1c-6044-4d67-9291-cb91c73b5207-attachment.attach"}, {"name": "请求参数", "source": "f5088c50-7770-4972-a902-1ebae0a3a515-attachment.attach"}], "start": 1617289579694, "stop": 1617289579699}, {"name": "不校验结果", "status": "passed", "start": 1617289579930, "stop": 1617289579930}], "start": 1617289579680, "stop": 1617289579930}], "start": 1617289579680, "stop": 1617289580025}